src/app/_components/GovernanceCalendarCard.tsx

```diff
@@ -0,0 +1,76 @@
+import { Clock, CalendarBlank } from '@phosphor-icons/react/dist/ssr'
+
+import { Heading } from '@/components/Heading'
+
+import { addMinutesToISO, extractTimeFromISO } from '@/utils/convertISOTime'
+
+import { TextLink } from './TextLink'
+
+export type GovernanceCalendarCardProps = {
+  title: string
+  startDate: string
+  cta: {
+    url: string
+    text: string
+  }
+  durationInMinutes: number
+  as?: React.ElementType
+}
+
+type CalendarProps = {
+  startDate: string
+}
+
+function Calendar({ startDate }: CalendarProps) {
+  const { day, month } = extractTimeFromISO(startDate)
+  return (
+    <div className="min-w-xs grid grid-rows-[_40px,_96px] rounded-md border border-blue-400 bg-blue-500 sm:min-w-32">
+      <span className="flex items-center justify-center rounded-tl-[5px] rounded-tr-[5px] text-base font-normal uppercase">
```

Comment:
I don't understand the reason behind `rounded-tl-[5px] rounded-tr-[5px]`. Why 5px?

Also, why not use `rounded-t-[value]` in one go?

Reply:
Ah you're right! Think `rounded-t-md` will do, for some reason it seemed off when I put the same value for outer and inner border, but on the second look it looks fine.